SPECS/python-click/python-click.spec

```diff
@@ -28,134 +20,219 @@ comes with good defaults out of the box.
 
 %description %{_description}
 
-%package -n     python%{python3_pkgversion}-%{pypi_name}
+%package -n     python3-%{pypi_name}
```

Comment:
This package should have a runtime requirement on `python3`

Reply:
Added.